src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -222,6 +222,10 @@ trait SubmissionComponent {
       filter(rec => rec.id === submissionId)
     }
 
+    def findBySubmitter(submitterId: String): SubmissionQueryType = {
+      filter(rec => rec.submitterId === submitterId)
```

Comment:
Right.

but by similar logic, `submitterId` is a Foreign Key already ... so doesn't it also already have an index?